### DISCUSSION TITLE
unable to use Stepper "TypeError: Cannot destructure property 'activateCallback' of 'undefined' as it is undefined."

### DISCUSSION THREAD

**Discussion by saikumar001-sky** [primefaces/community#2156] [PrimeVue]
<StepPanel v-slot="{ activateCallback }" value="1">
i'm using vue 3 and prime vue 4 version it throws error of destructure property 'activateCallback' of 'undefined'

**Comment by krepkyandriy** [comment]
you need to update to the latest version

**Comment by rwawiorka** [comment]
I encountered the same error, I managed to fix it through this snippet

```
<Step
  v-slot="{
    activateCallback = () => {},
    value = 1,
    a11yAttrs = {},
  } = {}"
  :value="1"
  asChild
>
```
